### PR TITLE
Replace Minimalism rules w/ Agile Manifesto values and principles

### PR DESCRIPTION
**Pull request by max-mironov**
Please follow https://github.com/Microsoft/code-with-engineering-playbook/issues/32 for further discussion

### CODE REVIEW

File: Engineering/BestPractices/AgileManifesto.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### We are uncovering better ways of developing software by doing it and helping others do it. Through this work we have come to value:
+
+* Individuals and interactions over processes and tools
+
+* Working software over comprehensive documentation
+
+* Customer collaboration over contract negotiation
+
+* Responding to change over following a plan
```
Comment (DanMass) [inline Engineering/BestPractices/AgileManifesto.md]: The Agile Manifesto has a copyright notice that we need to comply with, "© 2001, the above authors this declaration may be freely copied in any form, but only in its entirety through this notice." The above needs to include everything from http://agilemanifesto.org/, through the copyright notice, or just link to http://agilemanifesto.org/.

File: Engineering/BestPractices/AgileManifesto.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Principles behind the Agile Manifesto:
```
Comment (DanMass) [inline Engineering/BestPractices/AgileManifesto.md]: Why not just link to the principles at http://agilemanifesto.org/principles.html ?

File: Engineering/PracticiesAndPrinciples.md [hunk 3]
```diff
@@ -26,4 +26,4 @@ While working on the project we try to stick with best practices in architecture
 * [Avoid Technical Debt](https://martinfowler.com/bliki/TechnicalDebt.html)
 * [Use Done-Done checklist](BestPractices/DoneDone.md)
 * [Deliver using Minimal Valuable Slices](BestPractices/MinimalSlices.md)
-* [Follow Minimalism Rules](BestPractices/MinimalismRules.md)
```
Comment (DanMass) [inline Engineering/PracticiesAndPrinciples.md]: If we're removing the link to the MinimalismRules file, we should remove the file as well.

**Comment by fnocera** [comment]
@max-mironov The playbook has undergone a facelift and a restructure, I would love to work together to get this integrated in. Please rebase and update the PR given the new structure. Reach out to me if I can help at all. Thanks!

**Comment by max-mironov** [comment]
Hey @fnocera, in the light of the recent comments I'm not sure that we should merge current PR at all for now. Probably this one could be abandoned and we can discuss separately what specifically we could include here.

**Comment by fnocera** [comment]
@max-mironov as you suggested I will close this PR, we can then discuss offline.

**Pull request closed without merging**